src/App.tsx

```diff
@@ -1,16 +1,24 @@
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [copyOfMovies, setCopyOfMovies] = useState([...moviesFromServer]);
+
+  const onAddMoive = (newMovie: Movie) => {
```

Comment:
```suggestion
  const handleMovieAdd = (newMovie: Movie) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364

Reply:
Thank you, that was my weakness :) now is more clear for me.